geopandas/tests/test_cmapplotting.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from shapely.geometry import Point, MultiPoint
+from geopandas import GeoSeries, GeoDataFrame, plotting
+try:
+    if sys.version_info < (2, 7):
+        import unittest2
+    else:
+        raise ImportError()
+except ImportError:
+    import unittest
```

Comment:
For GeoPandas, we rely on the `pytest` framework for testing, and not on `unittest`. See https://docs.pytest.org/en/latest/ (see also the links we put in the notes of the meeting last week: https://hackmd.io/-o9Wcj85RteZzZzkk764VQ?both) 

In practice, this is not that a big difference though. One main difference is that you no longer subclass `unittest.TestCase` and that you can replace `self.assertEqual(a, b)` with `assert a == b` and things like that.

Reply:
@jorisvandenbossche I was trying to know whether the test works, but in doing so, I made a mistake. I will remove this commit right now.